xAPI.md

```diff
@@ -1943,8 +1943,9 @@ The table below lists all properties of the Attachment object.
 	<tr>
 		<td>sha2</td>
 		<td>String</td>
-		<td>The SHA-2 (SHA-256, SHA-384, SHA-512) hash of the attachment data. SHA-224 
-		SHOULD not be used: a minimum key size of 256 bits is recommended.</td>
+		<td>The SHA-2 (SHA-256, SHA-384, SHA-512) hash of the attachment data. <br/>
```

Comment:
I think the fact we list these three here in the type-that-is-required section might lead some to think SHA-224 isn't allowed at all. Propose we remove the list of three from here, and add to the below "The Client SHOULD use SHA-256, -384, or -512 for the "sha2" property"